gateway/src/apicast/policy/standalone/standalone.lua

```diff
@@ -319,13 +351,27 @@ end
 local function find_service(self, route)
   local destination = route and route.destination
 
-  if self.services and destination then
+  if not destination then return nil, 'no destination' end
+
+  if destination.service then
     return self.services[destination.service]
```

Comment:
now that you return `service, err` should return here `self.services[destination.service], nil`?

Reply:
The second return value is implicit nil. I don't see why it would be needed.